contributed_definitions/nyaml/NXstage_lab.yaml

```diff
@@ -121,7 +121,7 @@ NXstage_lab(NXcomponent_em):
     unit: NX_VOLTAGE
   # NEW ISSUE: rather the field if possible should be specified
   (NXpositioner):
-  
+  (NXsensor):
```

Comment:
NXstage_lab is quite similar to what we developed in MPES with [NXmanipulator](https://fairmat-nfdi.github.io/nexus_definitions/classes/contributed_definitions/NXmanipulator.html). Maybe could be harmonized at some point.

Reply:
I created an issue: https://github.com/FAIRmat-NFDI/nexus_definitions/issues/184